project.yaml

```diff
@@ -1,8 +1,13 @@
 version: "4.0"
 
 actions:
-  generate_dataset:
-    run: ehrql:v1 generate-dataset analysis/dataset_definition.py --output output/dataset.csv.gz
+  generate_measures_alt_tests:
+    run: >
+      ehrql:v1 generate-measures
+        analysis/measure_definition.py
+        --output output/alt_tests/measures.arrow
```

Comment:
The name of the sub-directory need not match the name of the codelist, because the action tells the reader which sub-directory is associated with which codelist. If a downstream action needs to associate a sub-directory with a codelist, then we may revisit this design decision.